<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffafcc;
border-color: rgb(0, 0, 0);
border: 2px solid black;</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>331</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-top-right-radius:50px;
background-color: #cdb4db;
 

   

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bauhaus 93</family>
       <pointsize>22</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #cdb4db;</string>
     </property>
     <property name="text">
      <string>Login In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsername">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>230</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border: 2px solid #555555;
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:0px;	</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPassword">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>300</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid #555555;
background-color:rgba(0,0,0,0);
border-bottom:2px solid rgba(46,82,101,200);
color:rgba(0,0,0,240);
padding-bottom:0px;	</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLogin">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>370</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #a2d2ff;
    border-radius: 15px;
    color: black;
    padding: 8px 16px;
}

QPushButton:hover {
	background-color: #bde0fe;
    text-decoration: underline;
}

QPushButton:pressed {
	background-color: #a2d2ff;
}</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>341</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../medicine.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #d5bdaf;
    border-radius: 15px;
    color: black;
    padding: 8px 16px;
}

QPushButton:hover {
	background-color: #e3d5ca;
	text-decoration: underline;
}

QPushButton:pressed {
	background-color: #d5bdaf;
}
</string>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
